models/model.js

```diff
@@ -73,3 +74,28 @@ exports.fetchUsers = () => {
     return users.rows;
   });
 };
+
+exports.createComment = (requestBody, article_id) => {
+  const { username, body } = requestBody;
+  if (article_id < 0 || article_id >= articles.length) {
```

Comment:
This could be done by querying the database to see if that article actually exists?